netbox/utilities/choices.py

```diff
@@ -222,8 +223,8 @@ class ImportFormatChoices(ChoiceSet):
     YAML = 'yaml'
 
     CHOICES = [
-        (AUTO, 'Auto-detect'),
-        (CSV, 'CSV'),
-        (JSON, 'JSON'),
-        (YAML, 'YAML'),
+        (AUTO, _('Auto-detect')),
+        (CSV, _('CSV')),
```

Comment:
Again, do we want to support translating file formats ?

Reply:
I've made the presumption elsewhere that acronyms like JSON, YAML, CSV, and so on do _not_ need to be translated. In this specific case, we'd want to translate the "auto-detect" option however.

Reply:
I agree, my comment started on L227+